orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -74,6 +83,62 @@ def _onnx_value_info_to_buffer_tensor(value_info, device):
     dtype = _utils.dtype_onnx_to_torch(value_info.type.tensor_type.elem_type)
     return torch.zeros(shape, device=device, dtype=dtype)
 
+def _parse_inputs_for_onnx_export(module, inputs):
```

Comment:
Similar comment as the earlier PR: we could combine this function with the other one named: ```_extract_input_information```?

Reply:
Git rebase failed and lots of code were lost. Just reapplied changes manually